zerver/tests/test_decorators.py

```diff
@@ -546,6 +546,15 @@ def test_check_none_or(self):
         x = 'x'
         self.assertEqual(check_none_or(check_int)('x', x), 'x is not an integer')
 
+    def test_check_url(self):
+        # type: () -> None
+        url = "http://127.0.0.1:5002/"  # type: Any
```

Comment:
Nit: Are there any other kinds of URLs (beyond IP addresses) that would make sense to include in the test?